UKHomeOffice/sgmr-service
# SPCR-243 Remove actions that allow user to cancel a cancelled voyage

## Description
Currently if a voyage is of the state cancelled or precancelled, when a user views this voyage, the ‘check’ page has the option to submit, cancel and exit without saving. If a voyage is cancelled, these CTAs should not be visible. 

### To test

- Click on 'View existing voyage plans'
- Go to the cancelled tab
- (If there are no cancelled reports, make a report and cancel it)
- Click on a cancelled report
- Scroll to the bottom
- There should be no submit, cancel or exit without saving visible
- Drafts and Submitted reports should still have these CTAs (for now at least)

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
